### DISCUSSION TITLE
Reality Models Sample App

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3136] [Q&A]
Is there any sample application that uses the Reality Model API? 
I want to upload my models in Cesium 3d Tiles format directly to ContextShare using Reality Data API

**Comment by pmconne** [comment]
There are currently no samples. Did you check [the tutorial](https://developer.bentley.com/tutorials/realitydata-api-quick-start/)?

**Reply by hanan-commits-work** [reply]
Yes i am following the tutorial. It would be easier however, if there were any samples.

**Reply by pmconne** [reply]
I wholeheartedly agree. @Pascal-Cloutier @rbois-bentley

**Reply by Pascal-Cloutier** [reply]
I also agree. A developer from the Reality Modeling team has been tasked to create samples for the ContextCapture and Reality Data APIs.

**Reply by hanan-commits-work** [reply]
Thank you.

**Reply by hanan-commits-work** [reply]
in the Create Reality Data API, what is the purpose of rootDocument key? Is it the location on bentley cloud or for uploading the model from local storage?

**Reply by Pascal-Cloutier** [reply]
The rootDocument is used to specify which document in the Reality data needs to be opened by the iTwin Viewer. This is especially useful for multiple-file formats such as 3D Tiles.

**Reply by hanan-commits-work** [reply]
Hello. 
Can anyone answer the question in this discussion,
https://github.com/iTwin/itwinjs-core/discussions/3141
